Article/Article.js

```diff
@@ -1,29 +1,33 @@
-// Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+// Because classes are not hoised you will need to start your code at the bottom of the page.
+// Look for the comment "START HERE"
 
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement
```

Comment:
try taking this base this.domElement and renaming it to something that will be more related to the object you will be affecting.